tests/build.go

```diff
@@ -139,6 +140,35 @@ func testImageIDOutput(t *testing.T, sb integration.Sandbox) {
 	require.Equal(t, dgst, digest.Digest(md.ConfigDigest))
 }
 
+func testBuildMobyFromLocalImage(t *testing.T, sb integration.Sandbox) {
+	if sb.Name() != "docker" {
+		t.Skip("skipping test for non-docker workers")
+	}
+
+	// pull image
+	cmd := dockerCmd(sb, withArgs("pull", "-q", "busybox:latest"))
```

Comment:
This will probably step over the local mirrors support in tests that avoid new pulls for every test. We could either connect the local mirrors to dockerd conf as well or  maybe rewrite the test so it doesn't require pull and instead does a "build + tag" as a setup.

Reply:
> We could either connect the local mirrors to dockerd conf

It looks already handled afaik: https://github.com/moby/buildkit/blob/cb86cc619256af8b3e6c9191b094d280fd804f86/util/testutil/integration/dockerd.go#L88-L92